scripts/entrypoint.sh

```diff
@@ -67,6 +69,16 @@ configure () {
     fi
     # split on commas and store the result in create_dbs
     IFS=',' read -ra create_dbs <<< "$tmp_dbs"
+
+    farm_props=
+    if [[ -n "${MDB_FARM_PROPERTIES:-}" ]]; then
+       IFS=',' read -ra farm_props <<< "${MDB_FARM_PROPERTIES}"
+    fi
+
+    db_props=
+    if [[ -n "${MDB_DB_PROPERTIES:-}" ]]; then
+        IFS=',' read -ra db_props <<< "${MDB_DB_PROPERTIES}"
```

Comment:
Is the `if` needed? Doesn't `read` correctly split the empty string into a zero-length array?
Update: On my system it does.
But maybe it's not wise to count on such an edge case.
Anyway, I wanted to mention it.

Reply:
I put up two remarks above but I'm fine with merging this as-is.